timesys/high-level-triggers.py

```diff
@@ -588,6 +588,21 @@
             'SI-Fam:PS-SDP2:BCKPLN', 'SI-Fam:PS-SDP3:BCKPLN',
             ),
         },
+    'SI-01:TI-Mags-FFCorrs': {
+        'database': {
+            'Src': {'enums': ('OrbSI', 'Cycle')},
```

Comment:
I think it is not necessary to include `'OrbSI'` here.

Reply:
done!